content/doc/developer/extensibility/index.adoc

```diff
@@ -22,4 +22,30 @@ references:
   title: My class is missing descriptor
 - url: https://wiki.jenkins-ci.org/display/JENKINS/Extension+Point+for+Project+Views+Navigation
   title: Extension Point for Project Views Navigation # TODO make into guide or remove as obsolete with BO
+- url: https://github.com/jenkinsci/sample-extensions
+  title: Sample extension implementations
 ---
+
+Jenkins is designed with extensibility in mind, both internally (core) and with plugins.
+Extensibility in Jenkins is accomplished by combining the concepts discussed below.
+
+== Extension annotation
+
++jenkinsdoc:Extension[]+ is an annotation that allows Jenkins to discover classes implementing certain types, instantiate them, and register them in global lists of implementations of its supertypes and interfaces.
```

Comment:
@daniel-beck the use of the "its" towards the end of the sentence I think is what's confusing me, what is the "its"?